backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/v2/product/ProductRestController.java

```diff
@@ -53,6 +56,13 @@ public ProductRestController(final ProductRestService productRestService)
 		this.productRestService = productRestService;
 	}
 
+	@ApiOperation("Create or update products and corresponding Bpartner-products.")
```

Comment:
please consolidate this class with `ProductsRestController` and add the API-Docu there.